kne/testbed.yml

```diff
@@ -0,0 +1,4 @@
+username: admin
+password: admin
+topology: /usr/local/google/home/robjs/go/src/github.com/openconfig/magna/kne/arista_magna.textproto
+cli: /usr/local/google/home/robjs/go/bin/kne
```

Comment:
Are these paths the ones we actually want here?

Reply:
Good catch, no. Currently in `featureprofiles` this is created dynamically. I'll remove this file from this repo for the moment, and as part of an integration test set up something that creates this topology.